src/heap.h

```diff
@@ -5,28 +5,114 @@ template<class ValueType>
 class Heap
 {
 private:
-    int heapSize;
     Node<ValueType>* root;
+    int size;
```

Comment:
Ta nazwa to celowo było heapSize, bo size jest zarezerwowane w c++. Mi kompilator dawał warning za to.

Reply:
Okej, to zmieniam na tak jak było